app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "from@example.com"
```

Comment:
the # is making the line comment out, should I remove it, or call this.mailer_suffix?